### DISCUSSION TITLE
求求求求！

### DISCUSSION THREAD

**Discussion by srtyhvff** [share121/inter-knot#4835] [委托]
你们每次评论都要登陆吗，反正我是，这正不正常啊

**Comment by March-777th** [comment]
不想登陆的话就进设置里换成公用接口

**Reply by ZYL-Z** [reply]
有什么弊端吗

**Reply by fzdqz** [reply]
> 有什么弊端吗
公用接口可能会API超速<img src="https://bbs-static.miyoushe.com/static/2023/11/16/232b6fd4f34212bd5f8f854b821b51ca_8087743279879108307.png" alt="妮可-哈哈" width="75" >
